src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +16,22 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> resultWords = new ArrayList<>();
+        try {
+            List<String> inputLines = Files.readAllLines(Paths.get(fileName));
+            for (String string : inputLines) {
```

Comment:
```suggestion
            for (String line : inputLines) {
```

Reply:
Wrong push. I don't have it in my code :(

Reply:
Question: is it ok to split declaring
 ` List<String> inputLines = null;
        try {
            inputLines = Files.readAllLines(Paths.get(fileName));
        } catch (IOException e) {`
and put in the try block only one readAllLines?